src/components/KTabs/KTabs.vue

```diff
@@ -102,13 +102,13 @@ watch(() => props.modelValue, (newTabHash) => {
 
 .k-tabs {
   ul {
-    border-bottom: 1px solid var(--KTabsBottomBorderColor, var(--grey-300, color(grey-300)));
+    border-bottom: var(--kui-border-width-10, $kui-border-width-10) solid var(--KTabsBottomBorderColor, var(--grey-300, var(--kui-color-border-neutral-weak, $kui-color-border-neutral-weak)));
     display: flex;
-    font-size: 18px;
-    line-height: 20px;
+    font-size: var(--kui-font-size-50, $kui-font-size-50);
+    line-height: var(--kui-line-height-30, $kui-line-height-30);
```

Comment:
the same step value for line-height should be used as font size in most scenarios, so here, it should be `line-height-50`

Reply:
But what was the reason that previously the value for font-size/line-height was different everywhere and now we are using same values ?

Reply:
The design token step values don't equal each other.

For example:

```
kui-font-size-50 = 18px
kui-line-height-50 = 28px
```

Reply:
updated.